### DISCUSSION TITLE
Ability to Make Certain Pages Only Visible to Certain People.

### DISCUSSION THREAD

**Discussion by Mich884** [jgraph/drawio#4180] [Ideas]
For work, I have a workflow that encompasses the process from conception to final production. The only thing that is bugging me is the fact that sales people (tailored for the first 2 pages) also has access to the production pages (next 4 pages) and vice versa. The main desire for this is to bulster internal company privacy of our processes.

**Comment by davidjgraph** [comment]
What authenication mechanism is used to get permissions to the diagram file?

**Reply by Mich884** [reply]
I just use the 'share now' button on the top right to invite people to be able to view the whole file. The file itself is stored on the company Google Drive. I hope that answers your question.

**Reply by davidjgraph** [reply]
This wouldn't be possible with Google Drive. The permissions are per file.

**Reply by Mich884** [reply]
That's understandable. Could you advise on if there are other ways other than Google Drive for this feature to be possible?

**Reply by davidjgraph** [reply]
It would need our own authenication system, that's not something we could reasonably implement. It could only be done with 2 files.

**Reply by Mich884** [reply]
Hello again,

Sorry if this would essentially have the same issue but when I mistakenly thought that this feature would allow just the current tab to be accessible to others in the sharable link:

![image](https://github.com/jgraph/drawio/assets/82031761/577ca5af-3331-4c03-9ae4-41cb829862df)

Sadly, it seems that it shares all of the tabs in the whole file:

![image](https://github.com/jgraph/drawio/assets/82031761/d90e419a-24a8-4ff0-973c-5bfc1a1724fd)

I thought, if actually having a feature as simple as allowing only a single tab (or even better, allow a range or custom amount of tabs) to be sharable through a link would work? 

I'm a bit confused but also curious as to what that current circled tick-box actually does because it really does look like it's meant to do exactly as the desired feature that I'm describing.
